tools/downloader/converter.py

```diff
@@ -25,6 +25,25 @@
 import common
 
 
+def run_pre_convert(reporter, model, output_dir, args):
```

Comment:
Hmm... I don't think "custom convert script" really reflects the semantics here - that would imply that the script replaces the entire conversion sequence with custom code, which isn't the case.

I named it "pre-convert script", because it runs before the standard conversion process. I think `convert_to_supported_format` is a decent proposal, but a) it's a bit unwieldy, and b) the script doesn't necessarily have to do that. I think it's possible that in the future we might use it for other purposes, e.g. to edit a file that's already in a supported format, or to create some auxiliary file that's needed for conversion.